api/controllers/install/controller.go

```diff
@@ -0,0 +1,113 @@
+package install
```

Comment:
Unsure if it should be the controller, the installation manager or through the API but we're missing a set of tests that:
- Validates the defaults are returned in the `Get` config and that those defaults don't overlap previously stored values
  - Specially make sure we test out the Global VS Pod/Service CIDR logic
- Validates CIDRs are computed on set config
- Validates we can Get config, perform a change and Set config (and that change is persisted)
- Validation is exercised in both Get and Set config.

IMO I think it would be easier to test out the controller, with our install manager and a mock store.